modules/ROOT/pages/rtm-agent-config-yaml.adoc

```diff
@@ -0,0 +1,145 @@
+= Update Agent Configuration in mule-agent.yml
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: agent, runtime manager, mule, esb, servers, monitor, notifications, external systems, third party, get status, metrics
+
+_Enterprise Edition_
+
+image:logo-cloud-disabled.png[link="/runtime-manager/deployment-strategies", title="CloudHub"]
+image:logo-hybrid-active.png[link="/runtime-manager/deployment-strategies", title="Hybrid Deployment"]
+image:logo-server-active.png[link="/runtime-manager/deployment-strategies", title="Anypoint Platform Private Cloud Edition"]
+
+
+When you run `amc_setup` with parameters to install Runtime Manager agent, the agent configuration is written to the `$MULE_HOME/conf/mule-agent.yml` file. At startup, the agent reads its configuration from this file.
+
+You can further configure or change the agent configuration by manually adding or editing parameters in the `mule-agent.yml` file.
+
+Some agent components allow you to configure them during runtime. For information, see xref:administration-service.adoc[Administration Service].
+
+
+== mule-agent.yml File Structure
+
+The `mule-agent.yml` file has the following structure:
```

Comment:
Wow, the numbering of lists in AsciiDoc is weird. I would expect 1., then a., then i....
But we are stuck with what we are stuck with, I guess.
(We should almost never use the i. level, anyway.)
I wonder if a graphic would be cooler?